src/test/java/uk/gov/pay/adminusers/resources/UserResourceAuthenticationTest.java

```diff
@@ -51,7 +51,7 @@ public void shouldAuthenticateUser_onAValidUsernamePasswordCombination() throws
                 .body("disabled", is(false))
                 .body("_links", hasSize(1))
                 .body("service_roles[0].role.name", is("admin"))
-                .body("service_roles[0].role.permissions", hasSize(35)); //we could consider removing this assertion if the permissions constantly changing
+                .body("service_roles[0].role.permissions", hasSize(37));
```

Comment:
As discussed we were wondering why it is not `38` as the total amount of permissions in the database.
We have `users-global:create` permission here https://github.com/alphagov/pay-adminusers/blob/5aeaa9c5382585a01e5d6d3373b0b3bdf53e795c/src/main/resources/config/initdata/permissions.csv#L4
Which is actually not linked/added to `admin` role here https://github.com/alphagov/pay-adminusers/blob/5aeaa9c5382585a01e5d6d3373b0b3bdf53e795c/src/main/resources/config/initdata/role-permissions.csv#L24
and the tests are executed against `admin` role, that is why we have `-1` permission from the total ones.